README.md

```diff
@@ -95,3 +95,7 @@ other news.
  - [AdsCommon ChangeLog](https://github.com/googleads/google-api-ads-ruby/blob/master/ads_common/ChangeLog)
  - [AdWords library ChangeLog](https://github.com/googleads/google-api-ads-ruby/blob/master/adwords_api/ChangeLog)
  - [DFP library ChangeLog](https://github.com/googleads/google-api-ads-ruby/blob/master/dfp_api/ChangeLog)
+
+### Note about this repo:
```

Comment:
Can you use the same heading level as the rest of the file (##) and rename this section to "Contributing"